harvard-lts/spotlight-oaipmh-resources
# Update email template

Ref https://github.com/harvard-lts/CURIOSity/issues/227

Email notification looks something like this now:

```
Harvesting complete for demo

The demo set has just finished harvesting to the CURIOSity Demo Site exhibit. Items may still be in the process of indexing. See delayed_jobs for current status.
```

### Testing Instructions 

1. Login as an admin 
2. Create a new exhibit 
3. Harvest a set into the new exhibit
4. When harvest completes, confirm you get an email like the example above.
